<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibratePlotter</class>
 <widget class="QDialog" name="CalibratePlotter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Erweiterte Einstellungen</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>9</y>
     <width>761</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0" columnminimumwidth="0,0,0,0,0,150">
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>horizontal
(Stift)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>88</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Messabstand
[mm]</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>longitudinal
(Papier)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_5">
      <property name="maximumSize">
       <size>
        <width>132</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Bewegen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>mm/
Schritt</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Richtung</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Motoren</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>94</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Motorschritte</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>vertikal
(Stift)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leCalibrationDistanceX">
      <property name="maximumSize">
       <size>
        <width>92</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Horizontaler Messabstand für die Anzahl der Motorschritte
Ein Negatives Vorzeichen spiegelt die Plotausgabe an der vertikalen Achse</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="leCalibrationStepsX">
      <property name="maximumSize">
       <size>
        <width>94</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Anzahl der Motorschritte horizontal</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="leCalibrationDistanceY">
      <property name="maximumSize">
       <size>
        <width>92</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Longitudinaler Messabstand für die Anzahl der Motorschritte
Ein Negatives Vorzeichen spiegelt die Plotausgabe an der horizontalen Achse</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="leCalibrationDistanceZ">
      <property name="maximumSize">
       <size>
        <width>92</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Vertikaler Messabstand für die Anzahl der Motorschritte</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="leCalibrationStepsY">
      <property name="maximumSize">
       <size>
        <width>94</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Anzahl der Motorschritte longitudinal</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="leCalibrationStepsZ">
      <property name="maximumSize">
       <size>
        <width>94</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Anzahl der Motorschritte vertikal</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="labelAccuracyX">
      <property name="toolTip">
       <string>Schrittweite in mm</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="labelAccuracyY">
      <property name="toolTip">
       <string>Schrittweite in mm</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="labelAccuracyZ">
      <property name="toolTip">
       <string>Schrittweite in mm</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="moveLeft">
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Fährt den Motor um die eingestellte Anzahl der Schritte nach links</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/images/left.png</normaloff>:/images/left.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moveRight">
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Fährt den Motor um die eingestellte Anzahl der Schritte nach rechts</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/images/right.png</normaloff>:/images/right.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="4">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="moveBackward">
        <property name="toolTip">
         <string>Fährt den Motor um die eingestellte Anzahl der Schritte nach vorn</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/images/backward.png</normaloff>:/images/backward.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moveForward">
        <property name="toolTip">
         <string>Fährt den Motor um die eingestellte Anzahl der Schritte nach hinten</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/images/forward.png</normaloff>:/images/forward.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="4">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="moveUp">
        <property name="toolTip">
         <string>Fährt den Motor um die eingestellte Anzahl der Schritte nach oben</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/images/up.png</normaloff>:/images/up.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="moveDown">
        <property name="toolTip">
         <string>Fährt den Motor um die eingestellte Anzahl der Schritte nach unten</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/images/down.png</normaloff>:/images/down.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="5">
     <widget class="QComboBox" name="comboMotorX">
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Wählt die Motoransteuerung für diese Achse aus</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QComboBox" name="comboMotorY">
      <property name="toolTip">
       <string>Wählt die Motoransteuerung für diese Achse aus</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QComboBox" name="comboMotorZ">
      <property name="toolTip">
       <string>Wählt die Motoransteuerung für diese Achse aus</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="StopMoving">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Stopt alle Motoren</string>
   </property>
   <property name="text">
    <string>Stopp</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="DrawCalibrationRectangle">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Zeichnet ein Rechtek zur Kontrolle der Kalibrierung</string>
   </property>
   <property name="text">
    <string>Kalibrierzeichnung</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Stiftspize</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leOffsetToPaper">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Abstand vom Kalibriepunkt zur Auflage</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leOffsetToPlate">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Abstand vom Kalibriepunkt zur Platte</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leOffsetToMovingPos">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Abstand vom Kalibriepunkt zur Ruhelage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>zum Papier</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>zur Platte</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>zur Ruhelage</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Abstand [mm]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Dicke [mm]</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePenWidth">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Hier kann die Stiftdicke für Postskriptdateien angegeben werden</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPortAddress">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>230</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>190</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Parallel Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>230</y>
     <width>57</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboPorts">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>78</width>
     <height>0</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Wählt die Motoransteuerung für diese Achse aus</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>250</y>
     <width>57</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pfad:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPortPath">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>250</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CalibratePlotter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CalibratePlotter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
